public/pages/utils/anomalyResultUtils.ts

```diff
@@ -352,49 +362,49 @@ export const getBucketizedAnomalyResultsQuery = (
   return {
     size: 0,
     query: {
-        bool: {
-          filter: [
-            {
-              range: {
-                data_end_time: {
-                  gte: startTime,
-                  lte: endTime,
-                },
+      bool: {
+        filter: [
+          {
+            range: {
+              data_end_time: {
+                gte: startTime,
+                lte: endTime,
               },
             },
-            {
-              term: {
-                detector_id: detectorId,
-              },
+          },
+          {
+            term: {
+              detector_id: detectorId,
             },
-          ],
-        },
+          },
+        ],
+      },
     },
     aggs: {
-        bucketized_anomaly_grade: {
-          date_histogram: {
-            field: 'data_end_time',
-            fixed_interval: `${fixedInterval}m`,
-          },
-          aggs: {
-            top_anomaly_hits: {
-              top_hits: {
-                sort: [
-                  {
-                    anomaly_grade: {
-                      order: 'desc',
-                    },
+      bucketized_anomaly_grade: {
+        date_histogram: {
+          field: 'data_end_time',
+          fixed_interval: `${fixedInterval}m`,
+        },
+        aggs: {
+          top_anomaly_hits: {
+            top_hits: {
+              sort: [
+                {
+                  anomaly_grade: {
+                    order: 'desc',
                   },
-                ],
-                _source: {
-                  includes: RETURNED_AD_RESULT_FIELDS,
                 },
-                size: 1,
+              ],
+              _source: {
+                includes: RETURNED_AD_RESULT_FIELDS,
               },
+              size: 1,
```

Comment:
If try to get last 7 days AD result for HC detector, this query will return only the top 1 result even user choose top 10 ?

Reply:
Yes. Actually right now we are relying on raw data for HC detector. I think we should be able to disable getting bucketized data since we don't rely on it. will change it.